tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -508,6 +510,11 @@ public bool DiagnosticSourceEnabled
         /// </summary>
         internal bool IsDataStreamsMonitoringEnabled { get; }
 
+        /// <summary>
+        /// Gets or sets the number of days a log file can be kept until
+        /// </summary>
+        internal int LogFileRetentionDays { get; set; }
```

Comment:
I don't think this property is ever used since you are reading the environment variable directly in `DatadogLogging.CleanLogFiles()`: 
```csharp
var logDaysLimit = EnvironmentHelpers.GetEnvironmentVariable(ConfigurationKeys.LogFileRetentionDays);
```

Reply:
Yeah I believe this goes with your comment above, before I thought it would configurable on the code too but the only way to access the value I found was the one below:

`EnvironmentHelpers.GetEnvironmentVariable(ConfigurationKeys.LogFileRetentionDays);`

So I think it should be good to remove the TracerSettings related implementation in that case.